Changes

```diff
@@ -151,7 +151,7 @@ Working version
 - #12634: Simplify TSan backtrace bookkeeping upon raise
   (Olivier Nicole and Fabrice Buoro, review by Gabriel Scherer)
 
-* #12686: Some primitives had the wrong types to be callable from the bytecode
+- #12686: Some primitives had the wrong types to be callable from the bytecode
```

Comment:
@kayceesrk this seems to be an unvoluntary change, probably introduced by the rebase. Can you revert it?